commons/error_constants.py

```diff
@@ -0,0 +1,104 @@
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+"""All common error constants string from cortx-test."""
+
+# S3 errors for Cortx s3 and RGW S3
+
+RGW_ERR_LONG_OBJ_NAME = ("An error occurred (InvalidObjectName) when calling the PutObject "
+                         "operation: Unknown")
+CORTX_ERR_LONG_OBJ_NAME = ("An error occurred (KeyTooLongError) when calling the PutObject "
+                           "operation: Your key is too long.")
+RGW_ERR_WRONG_JSON = ("An error occurred (InvalidPart) when calling the CompleteMultipartUpload "
+                      "operation: Unknown")
+CORTX_ERR_WRONG_JSON = ("An error occurred (InvalidPart) when calling the CompleteMultipartUpload "
+                        "operation: One or more of the specified parts could not be found. The "
+                        "part might not have been uploaded, or the specified entity tag might not "
+                        "have matched the part's entity tag.")
+RGW_ERR_DUPLICATE_BKT_NAME = ("An error occurred (BucketAlreadyExists) when calling the "
+                              "CreateBucket operation: Unknown")
+CORTX_ERR_DUPLICATE_BKT_NAME = ("An error occurred (BucketAlreadyOwnedByYou) when calling"
+                                "the CreateBucket operation: Your previous request to create "
+                                "the named bucket succeeded and you already own it.")
+RGW_ERR_COPY_OBJ = ("An error occurred (InvalidRequest) when calling the CopyObject operation:"
+                    " The specified copy source is larger than the maximum allowable size for a "
+                    "copy source: 5368709120")
+CORTX_ERR_COPY_OBJ = ("An error occurred (InvalidRequest) when calling the CopyObject operation:"
+                      " The specified copy source is larger than the maximum allowable size for a "
+                      "copy source: 5368709120")
+RGW_ERR_COPY_OBJ_METADATA = ("An error occurred (InvalidRequest) when calling the CopyObject "
+                             "operation: This copy request is illegal because it is trying to "
+                             "copy an object to itself without changing the object's metadata, "
+                             "storage class, website redirect location or encryption attributes.")
+CORTX_ERR_COPY_OBJ_METADATA = ("An error occurred (InvalidRequest) when calling the CopyObject "
+                               "operation: This copy request is illegal because it is trying to "
+                               "copy an object to itself without changing the object's metadata, "
+                               "storage class, website redirect location or encryption attributes.")
+RGW_HEAD_OBJ_ERR = ("An error occurred (404) when calling the HeadObject operation: Not Found")
+CORTX_HEAD_OBJ_ERR = ("An error occurred (404) when calling the HeadObject operation: Not Found")
+
+ACCOUNT_ERR = "attempted to create an account that already exists"
```

Comment:
If these messages are S3 protocol specific only then write such in a comment. If they are prefixed by S3_ then it would be easily recognizable that a message text is s3 response error message.